autogpt/web.py

```diff
@@ -34,6 +34,7 @@ def scrape_text_with_selenium(url):
     options.add_argument(
         "user-agent=Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/112.0.5615.49 Safari/537.36"
     )
+    options.add_argument("--headless")
```

Comment:
better use headless=new
https://developer.chrome.com/articles/new-headless/#new-headless-in-selenium-webdriver

Reply:
what is the difference?

Reply:
This article explains the difference: https://developer.chrome.com/articles/new-headless/

Reply:
```
google-chrome --version
Google Chrome 112.0.5615.121
```

with --headless only :
```
google-chrome --headless
[0415/192703.534998:WARNING:bluez_dbus_manager.cc(247)] Floss manager not present, cannot set Floss enable/disable.
[0415/192703.566803:WARNING:sandbox_linux.cc(393)] InitializeSandbox() called with multiple threads in process gpu-process.
```
with --headless=new no error thrown